src/components/member-role-update/index.js

```diff
@@ -41,6 +42,22 @@ const MemberRoleUpdate = () => {
   };
 
   const renderPromoteButton = () => {
+    if (isUserMember) {
+      return (
+        <>
+          <button
+            className={classNames.moveToMember}
+            type="button"
+            onClick={() => archiveTheMember(selectedMember)}
+          >
+            Archive Member
+          </button>
+          <br />
+          <p>{updateStatus}</p>
+        </>
+      );
+    }
```

Comment:
we need only one button when the user is a member, so instead of repeating yourself, you can put a condition below where the promote to member button only shows up if the user is not a member, also can you change the text in the button to `archive user` if the user is not a member